modules/core/src/main/java/org/locationtech/jts/index/intervalrtree/SortedPackedIntervalRTree.java

```diff
@@ -68,7 +66,7 @@ public void insert(double min, double max, Object item)
     leaves.add(new IntervalRTreeLeafNode(min, max, item));
 	}
 	
-  private void init()
+  private synchronized void init()
```

Comment:
It looks like this is the key change for concurrency?

Reply:
yes.